Submitted By:            Igor Živković <contact@igor-zivkovic.from.hr>
Date:                    2013-10-11
Initial Package Version: 0.5.1
Upstream Status:         Unknown
Origin:                  Gentoo
Description:             Fixes building with the latest mdds.

diff -Naur liborcus-0.5.1.orig/src/spreadsheet/sheet.cpp liborcus-0.5.1/src/spreadsheet/sheet.cpp
--- liborcus-0.5.1.orig/src/spreadsheet/sheet.cpp	2013-04-10 21:43:07.000000000 +0200
+++ liborcus-0.5.1/src/spreadsheet/sheet.cpp	2013-10-09 17:46:18.196256127 +0200
@@ -734,7 +734,7 @@
         con.build_tree();
 
     size_t index;
-    if (!con.search_tree(col, index))
+    if (!con.search_tree(col, index).second)
         return 0;
 
     return index;
